tests/test_graph.py

```diff
@@ -0,0 +1,19 @@
+from manim import *
+
+class PlotFunctions(GraphScene):
```

Comment:
I think this test is never being ran, right?

Reply:
Completly right. I totally forget that.